Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN echo "force-unsafe-io" > /etc/dpkg/dpkg.cfg.d/02apt-speedup \
 
 # Install prerequisites
 RUN set -x && \
-  apt-get update && apt-get -y upgrade && \
```

Comment:
I don't fully agree with the 'don't run apt-get upgrade' as it means basically run (potentially) with old libraries.  From a security perspective, it's potentially dangerous, and relying on updates from upstream.  Doing a dist-upgrade is silly though to do in docker though.